packages/expo-eas-client-id/ios/EASClientId/EASClientId.swift

```diff
@@ -0,0 +1,26 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import Foundation
+
+@objc
+public class EASClientId : NSObject {
```

Comment:
It's not Kotlin 😄 In Swift this 👇 is more common

```suggestion
public class EASClientId: NSObject {
```